akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +58,11 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    CompletionStage<Done> ignoreDataBytesJ(Materializer materializer);
```

Comment:
They would both have to be `discardDataBytes`.

The reason for me not liking `dataBytes` is that Bytes are data, do dataBytes doesn't really add any information. Isn't it more like `response.entity.source` or `response.entity.byteSource`? Anyway, that train has left the station.